luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -409,8 +421,48 @@ public Builder(LuthierResourceDictionaries resourceDictionaries, GranularityDict
             return logicalTableFactoryMap;
         }
 
-        public GranularityDictionary getGranularityDictionary() {
-            return granularityDictionary;
+        /**
+         * Default LogicalTable factories that are defined in fili-core.
+         *
+         * @return  a LinkedHashMap of MetricMaker name to its factory
+         */
+        private Map<String, Factory<LogicalMetric>> getDefaultMetricFactories() {
+            Map<String, Factory<LogicalMetric>> metricFactoryMap = new LinkedHashMap<>();
+            AggregationMetricFactory aggregationMetricFactory = new AggregationMetricFactory();
+            /* short aliases */
+            metricFactoryMap.put("aggregation", aggregationMetricFactory);
+            return metricFactoryMap;
+        }
+
+        /**
+         * Default LogicalTable factories that are defined in fili-core.
+         *
+         * @return  a LinkedHashMap of MetricMaker name to its factory
+         */
+        private Map<String, Factory<MetricMaker>> getDefaultMetricMakerFactories() {
+            Map<String, Factory<MetricMaker>> metricMakerFactoryMap = new LinkedHashMap<>();
+            ArithmeticMakerFactory arithmeticMakerFactory = new ArithmeticMakerFactory();
+            LongSumMakerFactory longSumMakerFactory = new LongSumMakerFactory();
+            AggregationAverageMakerFactory aggregationAvgMakerFactory = new AggregationAverageMakerFactory();
+            /* short aliases */
+            metricMakerFactoryMap.put("arithmetic", arithmeticMakerFactory);
+            metricMakerFactoryMap.put("longSum", longSumMakerFactory);
+            metricMakerFactoryMap.put("avg", aggregationAvgMakerFactory);
+            metricMakerFactoryMap.put("average", aggregationAvgMakerFactory);
+            /* fully qualified class names */
+            metricMakerFactoryMap.put(
+                    "com.yahoo.bard.webservice.data.config.metric.makers.ArithmeticMaker",
+                    arithmeticMakerFactory
+            );
+            metricMakerFactoryMap.put(
+                    "com.yahoo.bard.webservice.data.config.metric.makers.LongSumMaker",
+                    longSumMakerFactory
+            );
+            metricMakerFactoryMap.put(
+                    "com.yahoo.bard.webservice.data.config.metric.makers.AggregationAverageMaker",
+                    aggregationAvgMakerFactory
+            );
+            return metricMakerFactoryMap;
```

Comment:
as a general pattern, I would like to add className and short class name to these maker maps.  Perhaps we should make a utility method to do this for each of these that can look over the factories.

Reply:
This is done manually for now. I do see the value of having a helper function though. It can add clarity for sure but will need some string manipulation